README.md

```diff
@@ -110,8 +110,23 @@ https://2e1150434ba1d8c33bce7c82ee08b5d9850342c7@hookbot.scraperwiki.com/pub/foo
 The `HOOKBOT_KEY` should be kept secret and the authentication token is derived
 from it using a [HMAC](https://en.wikipedia.org/w/HMAC).
 
-Hookbot URLs
-------------
+Listening to multiple endpoints
+-------------------------------
+
+You can listen to all endpoints below a particular point in the hierarchy: ensure your
+path ends with `\` then append `?recursive`
+
+```
+hookbot make-tokens /sub/foo/?recursive
+ws://c0..9f@localhost:8080/sub/foo/%3Frecursive
+```
+
+Note that the output URL contains an URL-escape encoded question mark: this is *not* a query parameter.
+
+As shown above, the token is calculated *including* the `?recursive` option.
```

Comment:
Interesting. The resulting URL should not be escaped here, I think. Let's consider that one a bug. Can you fix this documentation `s/%3F/?/`, and I'll fix the code so that it doesn't do that.